src/golang/lib/storage/gcs.go

```diff
@@ -20,23 +22,44 @@ func newGCSStorage(gcsConfig *shared.GCSConfig) *gcsStorage {
 	}
 }
 
+// parseBucketAndKey uses a bucket in the form of bucket/path and a key and
+// returns the bucket name and the full key.
+func (g *gcsStorage) parseBucketAndKey(key string) (string, string) {
+	parts := strings.Split(g.gcsConfig.Bucket, "/")
```

Comment:
would it be easier to ask user to provide a `bucket` and a `path` separately? Or this is the canonical way to specify a gcs location?

Reply:
As of right now, we just allow specifying a bucket. The reason we need this method is for internal purposes because we want to put the vault in a special path under `/vault`. Any user facing changes would be out of scope for this PR.